distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestApplicationService.java

```diff
@@ -0,0 +1,246 @@
+
+    private static KarafConsole console;
+
+    private static final String APP_LIST_PROPERTIES_FILE =
+            "/org.codice.ddf.admin.applicationlist.properties";
+
+    @Inject
+    private ApplicationService applicationService;
+
+    @BeforeExam
+    public void beforeExam() throws Exception {
+        try {
+            basePort = getBasePort();
+            getAdminConfig().setLogLevels();
+            getServiceManager().waitForAllBundles();
+            console = new KarafConsole(bundleCtx, features, sessionFactory);
+        } catch (Exception e) {
+            LOGGER.error("Failed in @BeforeExam: ", e);
+            fail("Failed in @BeforeExam: " + e.getMessage());
+        }
+    }
+
+    /**
+     * Installs the configuration files needed at startup.
+     */
+    @Override
+    protected Option[] configureCustom() {
+        try {
+            return options(installStartupFile(getClass().getResourceAsStream(
+                    APP_LIST_PROPERTIES_FILE), "/etc" + APP_LIST_PROPERTIES_FILE));
+        } catch (Exception e) {
+            LOGGER.error("Could not copy app list to /etc directory", e);
+            return null;
+        }
+    }
+
+    @Test
+    public void aTestStartUpWithApplistPropertiesFile() throws Exception {
+        // Test AppService
+        getServiceManager().waitForRequiredFeature("admin-post-install-modules");
+        getServiceManager().waitForUninstallFeature("admin-modules-installer");
+        Application application = applicationService.getApplication(CONTENT_APP);
+        assertNotNull("Application [" + CONTENT_APP + "] must not be null after boot", application);
+        ApplicationStatus status = applicationService.getApplicationStatus(application);
+        assertThat("Application [" + CONTENT_APP + "] should be Started after boot",
+                status.getState(),
+                is(ACTIVE));
+
+        // Test Commands
+        String response = console.runCommand(STATUS_COMMAND + CONTENT_APP);
+        assertThat(CONTENT_APP + " should be active after boot",
+                response,
+                containsString(ACTIVE_APP));
+    }
+
+    @Test
+    public void bTestAppStatus() {
+        // Test AppService
+        Application catalog = applicationService.getApplication(CATALOG_APP);
+        assertNotNull("Application [" + CATALOG_APP + "] must not be null", catalog);
+        ApplicationStatus status = applicationService.getApplicationStatus(catalog);
+        assertThat("Application [" + CATALOG_APP + "] should be ACTIVE",
+                status.getState(),
+                is(ACTIVE));
+
+        Application solr = applicationService.getApplication(SOLR_APP);
+        assertNotNull("Application [" + SOLR_APP + "] must not be null", solr);
+        status = applicationService.getApplicationStatus(solr);
+        assertThat("Application [" + SOLR_APP + "] should be INACTIVE",
+                status.getState(),
+                is(INACTIVE));
+
```

Comment:
Updated this method to get the apps from the the `getApplications()` method instead.